Extensions/TweenBehavior/JsExtension.js

```diff
@@ -66,7 +66,7 @@ module.exports = {
       })
     );
 
-    extension
+   const tb = extension
```

Comment:
Nitpicking: avoid abbreviation, prefer full length variable name. Why? Because this helps refactoring and navigating the code. For example, I might do a refactoring in the whole codebase and search all actions and conditions declared on behaviors by searching for `behavior.add` and `Behavior.add` (to match things like `behavior.addAction` and `tweenBehavior.addAction`). But I would never search for `tb.add` ;)

When in doubt prefer a full variable name.